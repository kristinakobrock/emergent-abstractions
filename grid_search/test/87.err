  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:01, 24.58it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 23.35it/s] 18%|################7                                                                            | 9/50 [00:00<00:01, 25.47it/s] 24%|######################                                                                      | 12/50 [00:00<00:01, 26.52it/s] 30%|###########################6                                                                | 15/50 [00:00<00:01, 25.66it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 24.72it/s] 42%|######################################6                                                     | 21/50 [00:00<00:01, 24.45it/s] 48%|############################################1                                               | 24/50 [00:00<00:01, 23.39it/s] 54%|#################################################6                                          | 27/50 [00:01<00:00, 24.54it/s] 60%|#######################################################2                                    | 30/50 [00:01<00:00, 23.44it/s] 66%|############################################################7                               | 33/50 [00:01<00:00, 21.94it/s] 72%|##################################################################2                         | 36/50 [00:01<00:00, 21.82it/s] 78%|#######################################################################7                    | 39/50 [00:01<00:00, 23.57it/s] 86%|###############################################################################1            | 43/50 [00:01<00:00, 26.19it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 24.88it/s]100%|############################################################################################| 50/50 [00:02<00:00, 26.35it/s]100%|############################################################################################| 50/50 [00:02<00:00, 24.63it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 28.84it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 26.53it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 23.84it/s] 92%|####################################################################################9       | 12/13 [00:00<00:00, 23.21it/s]100%|############################################################################################| 13/13 [00:00<00:00, 25.20it/s]
