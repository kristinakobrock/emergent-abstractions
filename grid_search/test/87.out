# ["--attributes=5", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=64", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7359871864318848, "acc": 0.5818355679512024, "length": 3.0201499462127686, "mode": "train", "epoch": 1}
{"loss": 0.6992197632789612, "acc": 0.48496100306510925, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6699619293212891, "acc": 0.6170587539672852, "length": 1.1458393335342407, "mode": "train", "epoch": 2}
{"loss": 0.6680701971054077, "acc": 0.6244140267372131, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6604477167129517, "acc": 0.628491997718811, "length": 1.0403013229370117, "mode": "train", "epoch": 3}
{"loss": 0.6563572287559509, "acc": 0.6353515982627869, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6572632193565369, "acc": 0.6298035383224487, "length": 1.030738115310669, "mode": "train", "epoch": 4}
{"loss": 0.6552541255950928, "acc": 0.6353515982627869, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6578617691993713, "acc": 0.6297963261604309, "length": 1.0251986980438232, "mode": "train", "epoch": 5}
{"loss": 0.6553322076797485, "acc": 0.635546863079071, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.656886100769043, "acc": 0.6297959089279175, "length": 1.0223513841629028, "mode": "train", "epoch": 6}
{"loss": 0.6560974717140198, "acc": 0.6353515982627869, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6586107015609741, "acc": 0.6297323107719421, "length": 1.0205646753311157, "mode": "train", "epoch": 7}
{"loss": 0.6550388932228088, "acc": 0.6353515982627869, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.656515896320343, "acc": 0.6297924518585205, "length": 1.0170589685440063, "mode": "train", "epoch": 8}
{"loss": 0.6549351811408997, "acc": 0.6353515982627869, "length": 1.0, "mode": "test", "epoch": 8}
{"loss": 0.6567086577415466, "acc": 0.6297999024391174, "length": 1.0148608684539795, "mode": "train", "epoch": 9}
{"loss": 0.6544942855834961, "acc": 0.635546863079071, "length": 1.0, "mode": "test", "epoch": 9}
{"loss": 0.6566347479820251, "acc": 0.6297261714935303, "length": 1.0127665996551514, "mode": "train", "epoch": 10}
{"loss": 0.6550264954566956, "acc": 0.635546863079071, "length": 1.0, "mode": "test", "epoch": 10}
{"loss": 0.6556683778762817, "acc": 0.6300775408744812, "length": 1.0107277631759644, "mode": "train", "epoch": 11}
{"loss": 0.6555092930793762, "acc": 0.6353515982627869, "length": 1.0, "mode": "test", "epoch": 11}
{"loss": 0.6562809348106384, "acc": 0.629787027835846, "length": 1.0102580785751343, "mode": "train", "epoch": 12}
{"loss": 0.6553497910499573, "acc": 0.635546863079071, "length": 1.0, "mode": "test", "epoch": 12}
{"loss": 0.6561312675476074, "acc": 0.6299147009849548, "length": 1.0098376274108887, "mode": "train", "epoch": 13}
{"loss": 0.6555887460708618, "acc": 0.6351562142372131, "length": 1.0, "mode": "test", "epoch": 13}
{"loss": 0.6578823328018188, "acc": 0.6296525597572327, "length": 1.009572148323059, "mode": "train", "epoch": 14}
{"loss": 0.6563325524330139, "acc": 0.6351562142372131, "length": 1.0, "mode": "test", "epoch": 14}
{"loss": 0.6558311581611633, "acc": 0.6297917366027832, "length": 1.0099760293960571, "mode": "train", "epoch": 15}
{"loss": 0.6548663377761841, "acc": 0.6353515982627869, "length": 1.0, "mode": "test", "epoch": 15}
{"loss": 0.6594977974891663, "acc": 0.626815915107727, "length": 1.010485053062439, "mode": "train", "epoch": 16}
{"loss": 0.6581504344940186, "acc": 0.6333984732627869, "length": 1.0, "mode": "test", "epoch": 16}
{"loss": 0.6587199568748474, "acc": 0.6280877590179443, "length": 1.00790536403656, "mode": "train", "epoch": 17}
{"loss": 0.6561433672904968, "acc": 0.6353515982627869, "length": 1.0, "mode": "test", "epoch": 17}
{"loss": 0.6584272980690002, "acc": 0.6296944618225098, "length": 1.010145902633667, "mode": "train", "epoch": 18}
{"loss": 0.6581540107727051, "acc": 0.6351562142372131, "length": 1.0, "mode": "test", "epoch": 18}
{"loss": 0.6545771360397339, "acc": 0.6298412084579468, "length": 1.0117909908294678, "mode": "train", "epoch": 19}
{"loss": 0.658772349357605, "acc": 0.6353515982627869, "length": 1.0, "mode": "test", "epoch": 19}
{"loss": 0.6562944650650024, "acc": 0.6283156275749207, "length": 1.017636775970459, "mode": "train", "epoch": 20}
{"loss": 0.6567533612251282, "acc": 0.635546863079071, "length": 1.0, "mode": "test", "epoch": 20}
{"loss": 0.6528477668762207, "acc": 0.6302001476287842, "length": 1.1515415906906128, "mode": "train", "epoch": 21}
{"loss": 0.634708046913147, "acc": 0.6453858017921448, "length": 1.765625, "mode": "test", "epoch": 21}
{"loss": 0.6238535642623901, "acc": 0.653236985206604, "length": 1.8603423833847046, "mode": "train", "epoch": 22}
{"loss": 0.6291929483413696, "acc": 0.6371062994003296, "length": 1.7734375, "mode": "test", "epoch": 22}
{"loss": 0.6102961301803589, "acc": 0.6678294539451599, "length": 1.7277754545211792, "mode": "train", "epoch": 23}
{"loss": 0.6347822546958923, "acc": 0.6296783089637756, "length": 1.55078125, "mode": "test", "epoch": 23}
{"loss": 0.6090888381004333, "acc": 0.6652042865753174, "length": 1.778571367263794, "mode": "train", "epoch": 24}
{"loss": 0.6275672912597656, "acc": 0.6476989388465881, "length": 1.90234375, "mode": "test", "epoch": 24}
{"loss": 0.6036067008972168, "acc": 0.6699371933937073, "length": 1.8368566036224365, "mode": "train", "epoch": 25}
{"loss": 0.6286548376083374, "acc": 0.6387847065925598, "length": 1.90234375, "mode": "test", "epoch": 25}
{"loss": 0.5963531136512756, "acc": 0.6742362976074219, "length": 2.003736972808838, "mode": "train", "epoch": 26}
{"loss": 0.6270911693572998, "acc": 0.6422882080078125, "length": 1.73046875, "mode": "test", "epoch": 26}
{"loss": 0.591089129447937, "acc": 0.6820663213729858, "length": 2.2153351306915283, "mode": "train", "epoch": 27}
{"loss": 0.6052820086479187, "acc": 0.664599597454071, "length": 2.48828125, "mode": "test", "epoch": 27}
{"loss": 0.5783050656318665, "acc": 0.6923744082450867, "length": 2.4729199409484863, "mode": "train", "epoch": 28}
{"loss": 0.6001818180084229, "acc": 0.6812896728515625, "length": 2.81640625, "mode": "test", "epoch": 28}
{"loss": 0.563593327999115, "acc": 0.7076476216316223, "length": 2.7803001403808594, "mode": "train", "epoch": 29}
{"loss": 0.5789825916290283, "acc": 0.7020782232284546, "length": 3.15625, "mode": "test", "epoch": 29}
{"loss": 0.5445383191108704, "acc": 0.7236194014549255, "length": 3.1422345638275146, "mode": "train", "epoch": 30}
{"loss": 0.5590817928314209, "acc": 0.7108062505722046, "length": 3.34375, "mode": "test", "epoch": 30}
{"loss": 0.5141753554344177, "acc": 0.7455033659934998, "length": 3.2333080768585205, "mode": "train", "epoch": 31}
{"loss": 0.5626792311668396, "acc": 0.7284576296806335, "length": 3.44921875, "mode": "test", "epoch": 31}
{"loss": 0.49977001547813416, "acc": 0.7549945116043091, "length": 3.3886144161224365, "mode": "train", "epoch": 32}
{"loss": 0.5357292890548706, "acc": 0.7364348769187927, "length": 3.4140625, "mode": "test", "epoch": 32}
{"loss": 0.4750969409942627, "acc": 0.7672116756439209, "length": 3.372356653213501, "mode": "train", "epoch": 33}
{"loss": 0.5453641414642334, "acc": 0.735931396484375, "length": 3.32421875, "mode": "test", "epoch": 33}
{"loss": 0.45343735814094543, "acc": 0.782636284828186, "length": 3.3871500492095947, "mode": "train", "epoch": 34}
{"loss": 0.566868245601654, "acc": 0.7225494384765625, "length": 3.4765625, "mode": "test", "epoch": 34}
{"loss": 0.4420599937438965, "acc": 0.7944610118865967, "length": 3.4163687229156494, "mode": "train", "epoch": 35}
{"loss": 0.519476056098938, "acc": 0.7488097548484802, "length": 3.40234375, "mode": "test", "epoch": 35}
{"loss": 0.42045870423316956, "acc": 0.8016324639320374, "length": 3.4901328086853027, "mode": "train", "epoch": 36}
{"loss": 0.5332112312316895, "acc": 0.7506835460662842, "length": 3.54296875, "mode": "test", "epoch": 36}
{"loss": 0.4064640402793884, "acc": 0.8126002550125122, "length": 3.546116352081299, "mode": "train", "epoch": 37}
{"loss": 0.494075745344162, "acc": 0.769427478313446, "length": 3.5703125, "mode": "test", "epoch": 37}
{"loss": 0.39212021231651306, "acc": 0.8203217387199402, "length": 3.5853989124298096, "mode": "train", "epoch": 38}
{"loss": 0.4683316946029663, "acc": 0.7907836437225342, "length": 3.6328125, "mode": "test", "epoch": 38}
{"loss": 0.3852579891681671, "acc": 0.8277283310890198, "length": 3.599069595336914, "mode": "train", "epoch": 39}
{"loss": 0.4757382869720459, "acc": 0.7894012331962585, "length": 3.6875, "mode": "test", "epoch": 39}
{"loss": 0.36244359612464905, "acc": 0.8391727209091187, "length": 3.6579766273498535, "mode": "train", "epoch": 40}
{"loss": 0.45836296677589417, "acc": 0.7868958115577698, "length": 3.65625, "mode": "test", "epoch": 40}
{"loss": 0.3385448157787323, "acc": 0.851789653301239, "length": 3.722217082977295, "mode": "train", "epoch": 41}
{"loss": 0.42710572481155396, "acc": 0.8120453357696533, "length": 3.68359375, "mode": "test", "epoch": 41}
{"loss": 0.32491302490234375, "acc": 0.8595991134643555, "length": 3.72016978263855, "mode": "train", "epoch": 42}
{"loss": 0.43796610832214355, "acc": 0.8092193603515625, "length": 3.8125, "mode": "test", "epoch": 42}
{"loss": 0.323569118976593, "acc": 0.8605078458786011, "length": 3.759582757949829, "mode": "train", "epoch": 43}
{"loss": 0.4045841097831726, "acc": 0.826367199420929, "length": 3.765625, "mode": "test", "epoch": 43}
{"loss": 0.297978013753891, "acc": 0.871931254863739, "length": 3.7839901447296143, "mode": "train", "epoch": 44}
{"loss": 0.40043503046035767, "acc": 0.8285308480262756, "length": 3.81640625, "mode": "test", "epoch": 44}
{"loss": 0.29295414686203003, "acc": 0.8804256319999695, "length": 3.832275390625, "mode": "train", "epoch": 45}
{"loss": 0.4252069890499115, "acc": 0.8189544677734375, "length": 3.8984375, "mode": "test", "epoch": 45}
{"loss": 0.2752474844455719, "acc": 0.8870072960853577, "length": 3.8847804069519043, "mode": "train", "epoch": 46}
{"loss": 0.40113896131515503, "acc": 0.8354827165603638, "length": 3.89453125, "mode": "test", "epoch": 46}
{"loss": 0.26813241839408875, "acc": 0.8935877680778503, "length": 3.8697524070739746, "mode": "train", "epoch": 47}
{"loss": 0.3984188437461853, "acc": 0.842926025390625, "length": 3.875, "mode": "test", "epoch": 47}
{"loss": 0.254188597202301, "acc": 0.8995774984359741, "length": 3.886585235595703, "mode": "train", "epoch": 48}
{"loss": 0.39449456334114075, "acc": 0.8391265273094177, "length": 3.8828125, "mode": "test", "epoch": 48}
{"loss": 0.244888037443161, "acc": 0.9077839255332947, "length": 3.8944756984710693, "mode": "train", "epoch": 49}
{"loss": 0.3752858340740204, "acc": 0.8553892970085144, "length": 3.90234375, "mode": "test", "epoch": 49}
{"loss": 0.23881438374519348, "acc": 0.9130885601043701, "length": 3.9224607944488525, "mode": "train", "epoch": 50}
{"loss": 0.3677116632461548, "acc": 0.8622497320175171, "length": 3.94140625, "mode": "test", "epoch": 50}
{"loss": 0.21847012639045715, "acc": 0.9213629364967346, "length": 3.964143991470337, "mode": "train", "epoch": 51}
{"loss": 0.3542657196521759, "acc": 0.8824219107627869, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.213980033993721, "acc": 0.9230095148086548, "length": 3.9779603481292725, "mode": "train", "epoch": 52}
{"loss": 0.33937081694602966, "acc": 0.8856200575828552, "length": 3.9921875, "mode": "test", "epoch": 52}
{"loss": 0.20161840319633484, "acc": 0.9269530177116394, "length": 3.979229211807251, "mode": "train", "epoch": 53}
{"loss": 0.3434034585952759, "acc": 0.87890625, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.19364602863788605, "acc": 0.9326725602149963, "length": 3.9874134063720703, "mode": "train", "epoch": 54}
{"loss": 0.3338717222213745, "acc": 0.8853515982627869, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.1865580826997757, "acc": 0.9355114102363586, "length": 3.9934048652648926, "mode": "train", "epoch": 55}
{"loss": 0.319232702255249, "acc": 0.8878906965255737, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.1808340847492218, "acc": 0.9359493851661682, "length": 3.9915435314178467, "mode": "train", "epoch": 56}
{"loss": 0.3358598053455353, "acc": 0.8869141340255737, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.17077751457691193, "acc": 0.9421368837356567, "length": 3.994314670562744, "mode": "train", "epoch": 57}
{"loss": 0.3129418194293976, "acc": 0.896484375, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.1596248894929886, "acc": 0.9455301761627197, "length": 3.993847370147705, "mode": "train", "epoch": 58}
{"loss": 0.30477312207221985, "acc": 0.8986326456069946, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.1668301373720169, "acc": 0.9433411359786987, "length": 3.995457172393799, "mode": "train", "epoch": 59}
{"loss": 0.3461955189704895, "acc": 0.8828124403953552, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.16700685024261475, "acc": 0.940659761428833, "length": 3.9958908557891846, "mode": "train", "epoch": 60}
{"loss": 0.31547945737838745, "acc": 0.8988280296325684, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.8098334670066833
